doc/releases/changelog-0.36.0.md

```diff
@@ -6,28 +6,64 @@
 
 <h4>Estimate errors in a quantum circuit 🧮</h4>
 
-* Added `error` method to `QuantumPhaseEstimation` template.
-  [(#5278)](https://github.com/PennyLaneAI/pennylane/pull/5278)
+* This version of PennyLane lays the foundation for error estimation. [(#5154)](https://github.com/PennyLaneAI/pennylane/pull/5154)[(#5464)](https://github.com/PennyLaneAI/pennylane/pull/5464)[(#5465)](https://github.com/PennyLaneAI/pennylane/pull/5465)
 
-* Added new `SpectralNormError` class to the new error tracking functionality.
-  [(#5154)](https://github.com/PennyLaneAI/pennylane/pull/5154)
-
-* The `qml.TrotterProduct` operator now supports error estimation functionality. 
-  [(#5384)](https://github.com/PennyLaneAI/pennylane/pull/5384)
+  * For example, track the spectral norm error of custom operations by using the new `SpectralNormError` class and the error tracking functionality in `qml.specs` and `qml.Tracker`.
 
+  ```python
+  import pennylane as qml
+  from pennylane.resource.error import ErrorOperation, SpectralNormError
+
+  class XWithError(ErrorOperation):
+      def __init__(self, error_val, wires):
+          self.error_val = error_val
+          super().__init__(wires=wires)
+      def error(self):
+          return SpectralNormError(self.error_val)
+      
+      @staticmethod
+      def compute_matrix(*params, **hyperparams):
+          return np.array([[0,1],[1,0]])
+      
+  dev = qml.device('default.qubit', wires=2)
+  @qml.qnode(dev)
+  def circuit():
+      XWithError(0.1,[0])
+      XWithError(0.2,[1])
+      return qml.state()
+  ```
```

Comment:
````suggestion
    ```python
    import pennylane as qml
    from pennylane.resource.error import ErrorOperation, SpectralNormError

    class XWithError(ErrorOperation):
        def __init__(self, error_val, wires):
            self.error_val = error_val
            super().__init__(wires=wires)
        def error(self):
            return SpectralNormError(self.error_val)

    dev = qml.device("default.qubit")

    @qml.qnode(dev)
    def circuit():
        XWithError(0.1,[0])
        XWithError(0.2,[1])
        return qml.state()
    ```
````
Note that devices generally no longer need to have the number of wires specified.